pkgs/by-name/en/envision-unwrapped/package.nix

```diff
@@ -40,11 +47,19 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-J1zctfFOyu+uLpctTiAe5OWBM7nXanzQocTGs1ToUMA=";
   };
 
+  patches = [
+    ./support-headless-cli.patch
```

Comment:
Any reason why you aren't fetching this patch directly from [your MR on GitLab](https://gitlab.com/gabmus/envision/-/merge_requests/99)?

Reply:
Generally, this shouldn't be done, because:

* Fetching it from the MR is problematic because if I update the MR in almost any way the hash will change; and
* Fetching it from the commit changes is problematic because if I force-push to the branch the commits will eventually be garbage collected

Reply:
In that case, we should probably wait for upstream to merge the changes.

Reply:
But the problem is that the changes are necessary for the tests to run, and it might take a while for the changes to get merged because upstream doesn't seem to understand why they're needed. I'm not a fan of including the patch, but I've tested it fairly well and I think it's better than having the dependencies break _again_.